### DISCUSSION TITLE
Projects beta: editing drafts no longer working

### DISCUSSION THREAD

**Discussion by der** [community/community#16136] [Projects and Issues] [answered]
Until recently, if you had a `Draft` in an organisation "beta" project you could edit it by clicking on the title of the draft card.

This no longer works (in Chrome and Firefox). Now selecting the title opens a new tab on the same organisation project but no edit modal appears.

Looked for a new menu entry for this with no joy and tried hidden affordances like double clicking the card or selecting the card and hitting return, also no joy.

[This is possibly related to a recent change in behaviour for non-draft entries. For a short while if you clicked the title of a non-draft card you got a modal showing the issue in short form (with the option to jump to the full repository issue view in a new tab). That was a nice feature, bringing projects beta closer to the behaviour of non-beta projects. However, that functionality also disappeared at around the same time as the editing bug appeared.]

**Comment by sam-kayg** [comment]
+1. For me this only happens on the "Board" view with **Projects Side Panel (Alpha)** disabled. Clicking a draft item using the "table" view still opens the sidebar and allows editing.

If I enable the **Projects Side Panel (Alpha)** from Feature Preview settings then everything works again

**Reply by jasononeil** [reply]
Thanks for the workaround!

**Reply by apinste** [reply]
Thank you!

**Reply by zeke** [reply]
Worked for me. Thanks for sharing.

**Comment by jasononeil** [comment]
This is super frustrating, would love to see a fix soon

**Reply by apinste** [reply]
Same here.

**Comment by der** [comment, marked as answer]
I've seen no official messaging about this but now working again for me.

**Reply by azenMatt** [reply]
👋 hi @der let me know if you are still having problems with this, it looks like everything is now back working

**Reply by der** [reply]
Yes, working for now thanks.

**Reply by der** [reply]
That was supposed to say "working for _me_ now thanks"! :)
